crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/OnrampPresenterCoordinator.kt

```diff
@@ -56,6 +67,52 @@ internal class OnrampPresenterCoordinator @Inject constructor(
     private val currentLinkAccount: LinkController.LinkAccount?
         get() = interactor.state.value.linkControllerState?.internalLinkAccount
 
+    private val onrampActivityResultLauncher: ActivityResultLauncher<VerifyKycActivityContractArgs> =
+        activity.registerForActivityResult(VerifyKycInfoActivityContract()) { result ->
+            when (val action = result.action) {
```

Comment:
re: moving logic to the interactor

Essentially this entire `when` block should be moved to `OnrampInteractor`, say method 
```kotlin
suspend fun onVerifyKycResult(
    result: VerifyKycActivityContractResult
): OnrampVerifyKycInfoResult
```